API.md

```diff
@@ -75,227 +80,250 @@ Lets you draw a Point feature.
-var featureId = Draw.add(feature);
-console.log(featureId);
-//=> 'random-string'
+var featureIds = draw.add(feature);
+console.log(featureIds);
+//=> ['some-random-string']
 ```
 
-Example with ID:
+Example with a specified feature id:
 
 ```js
-var feature = { type: 'Point', coordinates: [0, 0], id: 'unique-id' };
-var featureId = Draw.add(feature);
-console.log(featureId)
-//=> unique-id
+var feature = {
+  type: 'Feature',
+  properties: {},
+  geometry: { type: 'Point', coordinates: [0, 0], id: 'unique-id' }
+};
+var featureIds = draw.add(feature);
+console.log(featureIds)
+//=> ['unique-id']
 ```
 
 ---
-###`.get(String: featureId) -> Object`
+### `get(featureId: string): ?Feature`
 
-This method takes an ID returns a GeoJSON feature.
+Returns the GeoJSON feature in Draw with the specified id, or `undefined` if the id matches no feature.
 
 Example:
 
 ```js
-var id = Draw.add({ type: 'Point', coordinates: [0, 0] });
-console.log(Draw.get(id));
+var featureIds = draw.add({ type: 'Point', coordinates: [0, 0] });
+var pointId = featureIds[0];
+console.log(draw.get(pointId));
 //=> { type: 'Feature', geometry: { type: 'Point', coordinates: [0, 0] } }
 ```
 
 ---
-### `.getFeatureIdsAt(Object: point) -> [featureId, featuresId]`
+### `getFeatureIdsAt(point: { x: number, y: number }): Array<string>`
 
-This method takes an object with x and y and returns a list of
-features currently rendered by draws at that spot.
+Returns an array of feature ids for features currently rendered at the specified point.
 
-This is good for using mouse events to get information out of draw.
+Notice that the `point` argument requires `x`, `y` coordinates from pixel space, rather than longitude, latitude coordinates.
 
-x and y must be from from pixel space, not latitude and longitude.
+With this function, you can use the coordinates provided by mouse events to get information out of Draw.
 
 ```js
 var featureIds = Draw.getFeatureIdsAt(20, 20);
 console.log(featureIds)
 //=> ['top-feature-at-20-20', 'another-feature-at-20-20']
 ```
 ---
-### `.getSelectedIds() -> [featureId, featuresId]`
+### `getSelectedIds(): Array<string>`
 
-This method returns the feature ids for all features currently in a selected state. If no features are currently selected than it will return an empty array.
+Returns an array of feature ids for features currently selected.
 
 ---
-### `.getSelected() -> Object`
+### `getSelected(): FeatureCollection`
 
-This method returns a FeatureCollection of all the selected features. If nothing is selected, it will return an empty FeatureCollection.
+Returns a FeatureCollection of all the features currently selected.
 
 ---
-###`.getSelectedPoints() -> [Object: GeoJSON]`
+### `getSelectedPoints(): FeatureCollection`
 
-This method returns all vertices currently in a selected state in an array of GeoJSON points.
+Returns a FeatureCollection of Points representing all the vertices currently selected.
```

Comment:
So... this returns the selected vertices, not the selected points. I just realized how confusing this name could be. We should think about changing it. We should also look to see how this works in `simple_select` vs `direct_select` mode.